### ISSUE TITLE
Vanity Adresses

### ISSUE THREAD

**Issue by HashMapsData2Value** [algorand/algorand-wallet#8]
It would be cool if we could generate vanity addresses from within the app. Since it is only on the phone, we probably can't expect to be able to produce more than 3-4 initial letters in a reasonable amount of time (hours). This way users wouldn't have to generate it on a computer and then move.

Not a high priority at all.

**Comment by Zambito1** [comment]
In the meantime, I created a script to generate vanity addresses some time ago.

https://robbyzambito.me/posts/how-to-generate-a-vanity-algorand-address/

This can actually be run on an Android device inside Termux, just make sure ~/go/bin is in your $PATH. Not sure if it can be run on iOS. I recommend running this program on a desktop or laptop either way, especially when searching for addresses with more than a few letters.

**Comment by scholtz** [comment]
Great idea.. I have implemented it in a-wallet..

![image](https://user-images.githubusercontent.com/1223439/122655209-146a5080-d151-11eb-8882-03facfd5c192.png)

**Comment by GoldenRed** [comment]
Indeed, it's great that there are tools but always feels sketchy of course to be copy+pasting.
